Frameworks/Foundation/NSUndoManager.mm

```diff
@@ -14,18 +14,84 @@
 //
 //******************************************************************************
 
+#import <Starboard.h>
 #import <StubReturn.h>
 #import <Foundation/NSUndoManager.h>
+#import "_NSUndoManagerInternal.h"
 
-NSString* const NSUndoManagerGroupIsDiscardableKey = @"NSUndoManagerGroupIsDiscardableKey";
+NSString* const NSUndoManagerGroupIsDiscardableKey = @"NSUndoManagerGroupIsDiscardable";
+
+@implementation NSUndoManager {
```

Comment:
>implementation [](start = 1, length = 14)

This class has `retain` and `copy` properties but it does not have a `dealloc` :|